<testsuites id="" name="" tests="1" failures="1" skipped="0" errors="0" time="2.691255000000121">
<testsuite name="login.spec.js" timestamp="2024-07-17T15:41:42.995Z" hostname="chromium" tests="1" failures="1" skipped="0" time="1.377" errors="0">
<testcase name="Logging out" classname="login.spec.js" time="1.377">
<failure message="login.spec.js:33:5 Logging out" type="FAILURE">
<![CDATA[  [chromium] › login.spec.js:33:5 › Logging out ────────────────────────────────────────────────────

    ReferenceError: ß is not defined

      33 | test('Logging out', async ({page}) => {
      34 |   await page.goto('https://relbuild.nanitor.net/');
    > 35 |   await page.getByRole('textbox').fill('max+53276572386587236@nanitor.com');ß
         |                                                                             ^
      36 |   await page.getByRole('button', { name: 'Sign in' }).click();
      37 |   await expect.soft(page.locator('form')).toContainText('Password');
      38 |   await page.getByRole('textbox').fill('Password123!');

        at /Users/devico/Documents/Nanitor UI Tests/tests/login.spec.js:35:77
]]>
</failure>
</testcase>
</testsuite>
</testsuites>